packages/evm/contracts/Yaru.sol

```diff
@@ -1,70 +1,70 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 pragma solidity ^0.8.17;
 
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import { ReentrancyGuard } from "@openzeppelin/contracts/security/ReentrancyGuard.sol";
```

Comment:
I generally prefer not to use ReentrancyGuard, and instead to write code to specifically thwart reentrancy for the given scenario.

Do you see any issues with the previous code to prevent reentrancy?

Reply:
No but since we removed `sender`, i decided to use the oz one!